CHANGELOG.md

```diff
@@ -4,6 +4,60 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## Unreleased
+
+**Minor features**
+* #2736 Support adding columns to hypertables with compression enabled
+* #2926 Optimize cagg refresh for small invalidations
+
+**Bugfixes**
+* #2883 Fix join qual propagation for nested joins
+* #2908 Fix changing column type of clustered hypertables
+* #2942 Validate continuous aggregate policy
+
```

Comment:
Shouldn't you list all issues that are cherry-picked? I'm missing issue #2831 and issue #2770 (or PR #2893)